MANIFEST.in

```diff
@@ -1,11 +1,15 @@
-include AUTHORS.rst
```

Comment:
This file is gone.

Reply:
Is there a particular reason behind removing this?

Reply:
This file doesn't exist. Maybe I'm missing something but a manifest check could not find it.